app/Http/Controllers/ProjectsBillingController.php

```
@@ -75,4 +81,44 @@ public function getProjectedProgressBilling(ProjectedProgressBillingRequest $req
+        return FinalBillingProjectionResource::collection($result['projects'])
+            ->additional([
+                'status' => 'success',
+                'message' => 'Final billing projection loaded successfully',
+                'overall_total' => $result['overall_total'],
+            ]);
+    }
+    public function getReceivableBillings()
+    {
+        $result = Project::select('id', 'code', 'name', 'location', 'amount', 'ntp_date')
+            ->get();
+        $gross_gt = $result->sum('amount');
```

PHPMD (2.15.0)

The variable $gross_gt is not named in camelCase. (undefined)

(CamelCaseVariableName)